app/src/app/[lng]/[inventory]/InventoryResultTab/TopEmissionsWidget.tsx

```diff
@@ -38,23 +39,28 @@ const TitleAndSelector = ({ t }: { t: Function }) => {
 };
 
 
-const EmissionsTable = ({ topEmissions }: { topEmissions: TopEmission[] }) => {
+const EmissionsTable = ({ topEmissions, t }: { topEmissions: TopEmission[], t: TFunction }) => {
   return (
     <TableContainer my={4}>
       <Table variant="simple">
         <Thead>
           <Tr>
-            <Th sx={{ "font": "bold", color: "black" }}>Subsector</Th>
-            <Th sx={{ "font": "bold", color: "black" }}>Total Emissions (CO2eq)</Th>
-            <Th sx={{ "font": "bold", color: "black" }}>% of Emissions</Th>
+            <Th sx={{ "font": "bold", color: "black" }}>{t("subsector")}</Th>
+            <Th sx={{ "font": "bold", color: "black" }}>{t("total-emissions-CO2eq")}</Th>
+            <Th sx={{ "font": "bold", color: "black" }}>{t("%-of-emissions")}</Th>
           </Tr>
         </Thead>
         <Tbody>
           {(topEmissions || []).map((emission, index) => (
             <Tr key={index}>
               <Td>
-                <Text fontSize="xl">{emission.subsectorName}</Text>
-                <Text fontSize="sm" color={"gray"}>{emission.sectorName}</Text>
+                <Text
+                  className="font-['Poppins'] text-sm leading-5 tracking-[0.5px]"
```

Comment:
Use the font/ fontFamily property from ChakraUI (I think it's font="heading" or something similar, we use this in other parts of the codebase already) instead of overwriting this from Tailwind. This way if we change the UI font family globally this won't break.